en_ewt-ud-train.conllu

```diff
@@ -101408,7 +101408,7 @@
 # sent_id = newsgroup-groups.google.com_eHolistic_417b09016150421f_ENG_20051120_175800-0023
 # newpar id = newsgroup-groups.google.com_eHolistic_417b09016150421f_ENG_20051120_175800-p0006
 # text = Faster, more efficient, delivery to the body Learn More
-1	Faster	faster	ADJ	JJR	Degree=Cmp	6	amod	6:amod	SpaceAfter=No
+1	Faster	fast	ADJ	JJR	Degree=Cmp	6	amod	6:amod	SpaceAfter=No
 2	,	,	PUNCT	,	_	6	punct	6:punct	_
 3	more	more	ADV	RBR	_	4	advmod	4:advmod	_
```

Comment:
Should this have `Degree=Cmp`? Lots of RBR/RBS without a Degree, esp. "more" and "most".

Reply:
I guess the issue is that "more" and "most" are analytic degree modifiers, rather than adjectives/adverbs inflected for degree, so maybe it's OK that they stay RBR/RBS but without a Degree.

There are a few tokens of "first" as RBS—should be RB. (And I think all the instances of adverbial "first", "third", etc. should technically have `NumType=Ord`, but that's a different problem.)

Missing Degree:

```
en_ewt-ud-train.conllu:10	less	less	ADV	RBR	_	11	advmod	11:advmod	_
en_ewt-ud-train.conllu:60	Stronger	strong	ADV	RBR	_	59	advmod	59:advmod	SpaceAfter=No
en_ewt-ud-train.conllu:9	wider	wide	ADV	RBR	_	6	conj	5:advmod|6:conj:rather_than	_
en_ewt-ud-train.conllu:15	etter	well	ADV	RBR	Typo=Yes	13	advmod	13:advmod	CorrectForm=better
en_ewt-ud-train.conllu:11	bigger	big	ADV	RBR	_	12	advmod	12:advmod	_
en_ewt-ud-train.conllu:47	lo9nger	long	ADV	RBR	Typo=Yes	46	advmod	46:advmod	CorrectForm=longer
```